src/platform/site-wide/announcements/components/WelcomeToNewVAModal.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Modal from '@department-of-veterans-affairs/formation/Modal';
+import brandConsolidation from '../../../brand-consolidation';
+
+export default class WelcomeToNewVAModal extends React.Component {
+  static isEnabled() {
+    if (!brandConsolidation.isEnabled()) return false;
+
+    if (__BUILDTYPE__ === 'preview') return false;
```

Comment:
Why don't we want this to run in preview?

Reply:
Good call. Will fix.